hawkular-alerts-rest-tests/src/test/groovy/org/hawkular/alerts/rest/ActionsITest.groovy

```diff
@@ -333,9 +333,7 @@ class ActionsITest extends AbstractITestBase {
         assertEquals(1, resp.data.size())
 
         // ENABLE Trigger
-        testTrigger.setEnabled(true);
-
-        resp = client.put(path: "triggers/test-email-threshold", body: testTrigger)
+        resp = client.put(path: "triggers/enabled", query:[triggerIds:"test-email-threshold",enabled:true] )
```

Comment:
Yes. If we eliminate all tests when we enable/disable a trigger using the normal update endpoint, we might miss a explicit test for testing the previous (and currently used) behaviour.
So, those changes are not bad, but we should maintain some tests to enable/disable triggers (non-group and groups) using the current and previous whole trigger update.